recipes/sdl/all/conanfile.py

```diff
@@ -1,6 +1,11 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
```

Comment:
Need to update `required_conan_version = ">=1.45.0"` to `1.51.3`: https://docs.conan.io/en/latest/changelog.html#aug-2022